### PR TITLE
stable/traefik/Chart.yaml: Update Traefik image to v1.7.7

### PR DESCRIPTION
**Pull request by nreisbeck**
#### What this PR does / why we need it:
Updates Traefik image version.

#### Checklist
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Chart Version bumped

### CODE REVIEW

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.59.0
-appVersion: 1.7.6
+version: 1.60.0
+appVersion: 1.7.7
```
Comment (cpanato) [inline stable/traefik/Chart.yaml:4]: @nreisbeck you also need to update here: https://github.com/helm/charts/blob/b6d71590913454cc9e4c0c814cbee0ad8faebb61/stable/traefik/README.md and here https://github.com/helm/charts/blob/b6d71590913454cc9e4c0c814cbee0ad8faebb61/stable/traefik/values.yaml#L3

File: stable/traefik/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.59.0
-appVersion: 1.7.6
+version: 1.60.0
```
Comment (cpanato) [inline stable/traefik/Chart.yaml]: also should be
```suggestion
version: 1.59.1
```

File: stable/traefik/Chart.yaml (hunk 1 above)
Comment (nreisbeck) [reply stable/traefik/Chart.yaml:4]: Whoops! Fixed both issues now.

**Review by nreisbeck — COMMENTED** [review]
Addressed both issues.

**Comment by cpanato** [comment]
@nreisbeck still missing update the readme.md

File: stable/traefik/values.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Default values for Traefik
 image: traefik
-imageTag: 1.7.6
+imageTag: 1.7.7
```
Comment (davidkarlsen) [inline stable/traefik/values.yaml:3]: please update the readme too so that it is consistent

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**